src/main/java/org/helpberkeley/memberdata/WorkRequestHandler.java

```diff
@@ -250,6 +253,12 @@ static class WorkRequest extends Reply {
             this.destinationTopic = reply.requestTopic;
             this.disableDateAudit = false;
             this.version = null;
+            this.disableMemberLimitAudit = false;
+
+        }
+
+        public boolean disableMemberLimitAudit(){
+            return this.disableMemberLimitAudit;
```

Comment:
don't need this accessor, the boolean is not private, so you can check it with request.disableMemberLimitAudit - no parens